indexer/packages/compliance/src/geoblocking/restrict-countries.ts

```diff
@@ -1,29 +1,27 @@
 import { stats } from '@dydxprotocol-indexer/base';
 
 import config from '../config';
-import { isRestrictedCountry } from './util';
+import { GeoOriginHeaders, GeoOriginStatus } from '../types';
 
-export interface CountryHeaders {
-  'cf-ipcountry'?: string,
-}
-
-export function isRestrictedCountryHeaders(headers: CountryHeaders): boolean {
+export function isRestrictedCountryHeaders(headers: GeoOriginHeaders): boolean {
   if (config.INDEXER_LEVEL_GEOBLOCKING_ENABLED === false) {
     return false;
   }
 
-  const ipCountry: string | undefined = headers['cf-ipcountry'];
+  const geoStatus: string | undefined = headers['geo-origin-status'];
 
   if (
-    ipCountry === undefined ||
-    isRestrictedCountry(ipCountry)
+    geoStatus === undefined ||
+    geoStatus !== GeoOriginStatus.OK
   ) {
     stats.increment(
       `${config.SERVICE_NAME}.rejected_restricted_country`,
       1,
       undefined,
       {
-        country: String(ipCountry),
+        country: headers['geo-origin-country'] || '',
+        region: headers['geo-origin-region'] || '',
+        status: headers['geo-origin-status'] || '',
```

Comment:
nit: status shouldn't need the `||` operator since we check for a legitimate status